module-services/service-desktop/endpoints/deviceInfo/DeviceInfoEndpointCommon.cpp

```diff
@@ -1,15 +1,23 @@
-// Copyright (c) 2017-2022, Mudita Sp. z.o.o. All rights reserved.
+// Copyright (c) 2017-2023, Mudita Sp. z.o.o. All rights reserved.
 // For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
 
 #include <endpoints/deviceInfo/DeviceInfoEndpointCommon.hpp>
 #include <endpoints/message/Sender.hpp>
-
 #include <service-desktop/ServiceDesktop.hpp>
-
 #include <sys/statvfs.h>
 
 namespace sdesktop::endpoints
 {
+
+    namespace
+    {
+        /* After UDM there are system_a and system_b partitions, but only one of them
+         * is mounted in MuditaOS at the time. However, the value returned by the
+         * endpoint should take into account space of both of them. */
+        constexpr auto numberOfSystemPartitions = 2;
```

Comment:
Might be defined in the body of `DeviceInfoEndpointCommon::getStorageInfo()` as it is only used there ;)

Reply:
Done ;)